### PR TITLE
Convert nodes chapter of Pacemaker Explained to rst+sphinx

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens, can you review when you get a chance? No rush

**Comment by kgaillot** [comment]
retest this please

**Comment by kgaillot** [comment]
retest this please

File: doc/sphinx/Pacemaker_Explained/reusing-configuration.rst [hunk 1]
```diff
@@ -251,16 +251,16 @@ So if for one resource you have
...
 
+   .. code-block:: xml
+   
+      <rsc_location id="WebDB-connectivity" rsc="WebDB">
+         <rule id-ref="ping-prefer-rule"/>
+      </rsc_location>
+   
 .. important::
-
+
```
[the remark this answers is not in the document]
Comment (kgaillot) [reply doc/sphinx/Pacemaker_Explained/reusing-configuration.rst]: fixed

**Comment by clumens** [comment]
Looks good to me.

**Pull request merged**